README.md

```diff
@@ -1,29 +1,34 @@
+<p>&nbsp;</p>
+<p align="center">
+<img src="https://raw.githubusercontent.com/aurora-is-near/aurora-workspace/main/res/aurora-workspace-logo.svg" width=500>
```

Comment:
The URL responds with a 404 error. I would suggest placing the image in the repo.